### PR TITLE
Create reviewers.yml

### PR DESCRIPTION
**Pull request by ashish4321**
#20 Fixed

### CODE REVIEW

**Comment by ashish4321** [comment]
@DreamyPhobic Pls review it

File: .github/reviewers.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+reviewers:
+-DreamyPhobic
```
Comment (DreamyPhobic) [inline .github/reviewers.yml]: give space between hyphen and username.

File: .github/reviewers.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+-shiv07tiwari
```
Comment (DreamyPhobic) [inline .github/reviewers.yml]: same as stated above.

**Comment by DreamyPhobic** [comment]
Nice! Enough of the good first issues, now do some issues with more difficulty level... We will shortly add some more good issues.

**Comment by DreamyPhobic** [comment]
From next time, send PR according to the template.

**Comment by ashish4321** [comment]
okk

**Pull request merged**